templates/404.phtml

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <?php
+    require 'partials/layout/head.phtml';
+    echo \Portal\ViewHelpers\LayoutViewHelper::outputTitle('404');
+    ?>
+</head>
+<body>
+<?php require 'partials/layout/header.phtml'; ?>
+
+<?php require 'partials/layout/nav.phtml'; ?>
+<main>
+    <div class="container">
+        <div class="alert alert-info">
```

Comment:
not a fan of using an alert here. we use alerts for interaction responses (like success and error messages when submitting a form). build something bigger and more obvious that its a 404